### PR TITLE
Client only

### PR DESCRIPTION
**Pull request by mattjstar**
Check it out!

http://critic-belinda-75735.netlify.com/

We now have a netlify account, that we can have auto-deploy our site. For open source projects their service is free!

I'm going to close the heroku app setup PR (https://github.com/wework/careday-app/pull/26). We can always go back to it if we want to setup universal react with server rendering.

### CODE REVIEW

File: webpack.client.babel.js [hunk 1]
```diff
@@ -1,25 +1,22 @@
 // TODO
-// - setup production/deployment build
+// - Consolidate duplicate configuration from webpack.server and webpack.client into a base webpack file
```
Comment (nason) [inline webpack.client.babel.js]: Don't think we need to do this...for now. Do we?

File: webpack.client.babel.js [hunk 2]
```diff
@@ -1,25 +1,22 @@
 // TODO
-// - setup production/deployment build
```
Comment (nason) [inline webpack.client.babel.js]: This TODO is still pretty necessary and I opened #28 to track it

screen shot 2017-04-22 at 12 02 44 am

Comment (mattjstar) [inline webpack.client.babel.js]: Yes very good call!

File: webpack.client.babel.js (hunk 1 above)
Comment (mattjstar) [inline webpack.client.babel.js]: yep, accidentally left that in there, gonna update the note to point to https://github.com/wework/careday-app/pull/26 if we want to revisit server side rendering

**Pull request merged**